makefiles/vars.inc.mk

```diff
@@ -21,6 +21,7 @@ export RIOTBASE              # The root folder of RIOT. The folder where this ve
 export RIOTCPU               # For third party CPUs this folder is the base of the CPUs.
 export RIOTBOARD             # For third party BOARDs this folder is the base of the BOARDs.
 export RIOTPKG               # For overriding RIOT's pkg directory
+export RIOTTOOLS             # For overriding RIOT's tools directory
```

Comment:
Could this be done in a separate PR/issue, because right now there are still a lot of `dist/tools` in the repository, and only two packages using the variable does not help keeping things consistent:
```
git grep 'dist/tools' | wc -l
219
```

Reply:
well that search mostly hits `dist/tools` in comments and readme docs, nevertheless you got a point, that this is sub-optimal and inconsistent. I'll look into it.

Reply:
most hits are not actual usages but rather comments, e.g. in README.

Reply:
There are still a lot, all the python tests use it for example.

git grep dist/tools | grep -v -e README -e doc.txt -e.md | wc -l
176